dashboard/app/controllers/home_controller.rb

```diff
@@ -112,6 +112,7 @@ def terms_and_privacy
 
     current_user_permissions = UserPermission.where(user_id: current_user.id).pluck(:permission)
     @homepage_data[:showStudentAsVerifiedTeacherWarning] = current_user.student? && current_user_permissions.include?(UserPermission::AUTHORIZED_TEACHER)
+    @homepage_data[:showDeprecatedCalcAndEvalWarning] = current_user.student? && ProjectsList.user_has_project_type(current_user.id, ['algebra_game', 'calc', 'eval'])
```

Comment:
Will the deprecation warning not be displayed for teachers with these types of projects?

Reply:
What kind of delay would it have to add it for teachers as well?

Reply:
I think it's just tweaking the condition above.

Reply:
Yes, we can just remove the check for being a student, and then it should show for both students and teachers.

Reply:
Oh, this check is used for the user's homepage. If we want the notification to show on the teacher's homepage as well, we'll need to modify the client-side code that renders their homepage.

Reply:
I've updated the change to also show the notification on a teacher's `/home` and /projects` page.